Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -0,0 +1,197 @@
+﻿namespace BootCamp.Chapter1
+{
+    public static class ArrayOperations
+    {
+        /// <summary>
+        /// Checks if array is null or empty
+        /// </summary>
+        /// <param name="array"></param>
+        /// <returns>true or false</returns>
+        public static bool IsNullOrEmpty(int[] array)
+        {
+            if(array == null || array.Length == 0)
```

Comment:
You can directly `return  array == null || array.Length == 0` because it translate to true or false. Now it reads:
```
if(true) return true;
else return false;
```
which is redundant;